src/styles/zIndex.js

```diff
@@ -1,6 +1,7 @@
 export default {
   menu: 1000,
   appBar: 1100,
+  bottomNavigation: 1150,
```

Comment:
I don't the the visual button navigation should have a zindex logic. I would rather have a layouting wrapper to handle it.